src/Database.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PKP\Beacon;
+
+use Illuminate\Database\Capsule\Manager as Capsule;
+
+class Database
+{
+    /** @var Capsule */
+    protected $capsule;
+
+    /**
+     * Constructor
+     */
+    public function __construct()
+    {
+        $this->capsule = new Capsule();
+
+        $this->capsule->addConnection([
+            'driver' => 'mysql',
+            'host' => getenv('BEACON_DBHOST') ?: '192.168.0.105',
```

Comment:
Could you leave the default set with hostname `localhost` and database user `beacon`?